broken_link_finder/src/main/java/com/testsigma/addons/broken_link_finder/web/FindAllBrokenImagesInPageAndAllChildPages.java

```diff
@@ -90,7 +90,8 @@ void collectBrokenImages(String URL) {
         }
     }
 
-    void collectValidLinks(String URL, Integer nestedIterationsLevel) {
+    Boolean collectValidLinks(String URL, Integer nestedIterationsLevel) {
+        if(URL==null) return false;
```

Comment:
@shabarish-testsigma have log states so that it will be visible on test step

Reply:
@PratheepV  added the logs